accord-core/src/main/java/accord/api/Data.java

```diff
@@ -23,6 +23,15 @@
  */
 public interface Data
 {
+    Data NOOP_DATA = new Data()
+    {
+        @Override
+        public Data merge(Data data)
+        {
+            return NOOP_DATA;
```

Comment:
It's a good point that that we should handle that case, but is returning the other `Data` the right call here?

It is no op, not empty. Accord can't have an empty one because it is the integration that knows what a `Data` concretely is so if you attempt to mix no op with something else it's a bug.

I'll have it check that the incoming data is either null or also a no op data.